mkdocs/cli.py

```diff
@@ -120,13 +122,14 @@ def json_command(clean, config_file, strict, site_dir):
 @cli.command(name="gh-deploy")
 @click.option('--clean', is_flag=True, help=clean_help)
 @click.option('--config-file', type=click.File('rb'), help=config_file_help)
-def gh_deploy_command(config_file, clean):
+@click.option('--commit-message', is_flag=False, help=commit_message_help)
```

Comment:
I agree `--message` or `-m` is better than `--commit-message`. I've effected the change.